### PR TITLE
Create 2021-08-11-living_bot-paddleXwechaty.md

### PR DESCRIPTION
**Pull request by thomas-yanxin**
Please make sure to check the following boxes creating your pull request, thank you very much!

- [x] Add Blog Header, including title, author, image, categories, tags, etc.  

    For example:  

    ```md
    ---
    title: Your Great Blog Title
    author: your_github_id
    categories: project
    tags:
      - python
      - padlocal
    image: /assets/2021/02-your-blog-slug/teaser.jpg
    ---

    One line bio

    Your blog post content Your blog post content
    Your blog post content Your blog post content
    Your blog post content Your blog post content
    ```

    > It will be easier to start drafting your post by using a existing post file as your template, for example, [this one](https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_posts/2021-01-14-wechaty-puppet-service.md)

- [x] Keep all filenames lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor.md` instead of `2017-10-06-WeChat PC Impactor.md`
- [x] Embed video/pdf from YouTube/Bilibili/local (if there's any) by using `{% include iframe.html src="URL_or_PATH" %}`. (Learn more about how to embed file/url in the post by reading this [blog post](https://wechaty.js.org/2020/08/24/add-video-to-wechaty-blog/)
- [x] Save photo to local before publishing, to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2021/` directory.
- [x] Select a beautiful and meaningful teaser image for your blog
- [x] Create your contributor profile (if you are a first time contributor)

    For excample:

    > jekyll/_contributors/lijiarui.md

    ```yaml
    ---
    name: Jiarui LI (李佳芮)
    site: https://lijiarui.github.io
    bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
    avatar: /jekyll/assets/contributors/lijiarui/avatar.png
    email: rui@juzi.bot
    twitter: https://weibo.com/u/2175505900
    ---
    ```

  Here is the example file:

  - lijiarui.md: <https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_contributors/lijiarui.md>
  - It will shown like this on wechaty website: <https://wechaty.js.org/contributors/lijiarui/>

To learn more about the contribute guideline, visit: <https://wechaty.js.org/docs/contributor-program/>

### CODE REVIEW

**Review by huan — CHANGES_REQUESTED** [review]
Thanks for sharing your experiences with the Paddle + Wechaty event!

Please read the GitHub Actions CI messages and fix testing problems then we can continue reviewing this great PR.

**Comment by lijiarui** [comment]
@thomas-yanxin  Please sign the CLA first.

Looking forward to your reply.

**Comment by lijiarui** [comment]
Hi @thomas-yanxin Please 
- Add author info as well.
- Make Github action green.

Thanks

**Comment by lijiarui** [comment]
ping @thomas-yanxin 

Please always keep working on one PR and keep improving it, instead of closing one and opening another, because we should always keep the conversation and the improvements linked together, which can be achieved by keeping improving one PR.

I think you can commit https://github.com/wechaty/wechaty.js.org/pull/1191  to this PR.

File: jekyll/_posts/2021-08-11-living_bot-paddleXwechaty.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  - paddlepaddle
+  - wechaty
+---
+
+
+# 『PaddlePaddle X Wechaty』有颜又有才的Living_Bot
+
+## 视频奉上先
+
+<div class="video-container" style="
```
Comment (huan) [inline jekyll/_posts/2021-08-11-living_bot-paddleXwechaty.md]: Use `{% include iframe.html src="https://www.bilibili.com/video/BV1pf4y1V7G8/" %}` instead.

**Review by huan — CHANGES_REQUESTED** [review]
Please make the CI green before we can continue reviewing this great blog post!

**Comment by thomas-yanxin** [comment]
I feel very sorry for some inappropriate operations before, because I just learned about Git related operations, and I am not familiar with it. I'm still learning how to do it. I'm so sorry for that! For the above reasons, can you take a look at this PR #1270.

**Comment by thomas-yanxin** [comment]
Maybe I push my contributor profile and the blog together into this pr #1270.

**Comment by huan** [comment]
No problem. Please keep one PR which you are working on, and closed others to help us to be focus. 

Thank you very much.

**Pull request closed without merging**